### ISSUE TITLE
terraform --version has different standar output on different platforms.

### ISSUE THREAD

**Issue by serafeimgr** [hashicorp/terraform#26279]
The standar output is not consistent between platforms.

Examples below

OSx

```
terraform --version
Terraform v0.12.26

Your version of Terraform is out of date! The latest version
is 0.13.2. You can update by downloading from https://www.terraform.io/downloads.html
```

Linux

```
terraform --version
Your version of Terraform is out of date! The latest version
is 0.13.3. You can update by downloading from https://www.terraform.io/downloads.html
Terraform v0.13.0
```

this can lead to misunderstanding 

Kind regards,
Serafeim

**Comment by alisdair** [comment]
This is a bug in Terraform 0.13.0 which was fixed in #25811 and released as part of Terraform 0.13.1. Please try again with the latest Terraform release to see the fix.

**Comment by serafeimgr** [comment]
True, thank you.
